.changeset/real-spies-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/tailwind': major
```

Comment:
Mistake?

Reply:
I can't see the history of the file. @TomPichaud is this change **intentional**? Shouldn't this be a `patch` change, considering the PR says  "fix"? 

If this is a `major`, and breaks existing APIs, the changeset should mention a migration path for what's changed.

Reply:
I've updated this part with what changed and the migration path.